package.json

```diff
@@ -10,10 +10,10 @@
   },
   "scripts": {
     "start": "node .",
-    "dev": "parallelshell 'NODE_ENV=development nodemon .' 'npm run watch:scss' 'npm run watch:js' 'npm run watch:translations'",
-    "watch:scss": "nodemon -e scss -x 'npm run sass'",
-    "watch:js": "nodemon --watch assets/js -x 'npm run browserify'",
-    "watch:translations": "nodemon --watch apps/my_awesome_form/translations/src -x 'npm run hof-transpile'",
+    "dev": "parallelshell 'NODE_ENV=development ./node_modules/.bin/nodemon .' 'npm run watch:scss' 'npm run watch:js' 'npm run watch:translations'",
+    "watch:scss": "./node_modules/.bin/nodemon -e scss -x 'npm run sass'",
```

Comment:
yeah I don't think we need to be so explicit. NPM takes care of this so lets just assume people know that.

If you remove it kindly I shall merge like a mad man.